.changelog/6038.feature.md

```diff
@@ -0,0 +1 @@
+go/runtime/host/sgx: Retry re-attestation faster on failure
```

Comment:
This could also be trivial, instead of a feature.

Reply:
Changed.